### ISSUE TITLE
README's old harmonic oscillator example was good

### ISSUE THREAD

**Issue by alok** [lecopivo/SciLean#51]
Now it's buried (without pictures) in the reference book as an example, but it was that example that really showed me just how powerful the whole approach was and why it was a scientific computing *assistant*.

**Comment by lecopivo** [comment]
Yes yes I agree, I want to work on more examples, create pictures and clicking on those pictures will take you to explanations of each example.
